src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+
+  gap: $gap;
+  padding: $padding;
+
+  width: $width*7 + $padding*2 + $gap*6;
+
+  &__day {
+    @extend %day__style;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    transition-duration: 250ms;
+
+    &:hover {
+      transform: translate(10px,-5px);
+      background-color: pink;
+    }
+
+      @for $num from 0 to 32 {
+        &:nth-child(#{$num})::after {
+        content: "#{$num}";
+        }
+      }
+
```

Comment:
why from 0 to 32 ? ) it's just a question)

Reply:
Because, month has 31 days max, in this loop it counts from 0 to 32, not including 32.